app/controllers/items_controller.rb

```diff
@@ -619,9 +622,9 @@ def add_workflow
 
   private
 
+  # XXX: note that this logic is replicated in DorController#reindex. probably should be moved to a helper
   def reindex item
-    doc=item.to_solr
-    Dor::SearchService.solr.add(doc, :add_attributes => {:commitWithin => 1000})
+    Dor::SearchService.solr.add(item.to_solr, :add_attributes => {:commitWithin => 1000})
```

Comment:
Move it to a helper in this PR or create an issue for it?  The comment might get stale unless we create an action item for it.